examples/cfrl_adult.ipynb

```diff
@@ -0,0 +1,1449 @@
+{
```

Comment:
<p>"uses an extrac type checking"  ?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/SeldonIO/alibi/pull/457/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
should have been `extra`. But maybe `additional` sounds better.